brands/amenity/fast_food.json

```diff
@@ -251,6 +251,37 @@
       "takeaway": "yes"
     }
   },
+  "amenity/fast_food|Burgerim": {
+    "countryCodes": ["il"],
+    "tags": {
+      "amenity": "fast_food",
+      "brand": "Burgerim",
+      "brand:en": "Burgerim",
+      "brand:he": "בורגרים",
+      "brand:wikidata": "Q64760354",
+      "cuisine": "burger",
+      "name": "Burgerim",
```

Comment:
`name` and `brand` for some of these entries is in English but for other entries is in Hebrew. Is that consistent with how these POIs are mapped (or should be mapped) in Israel in OSM?

Reply:
Some names are obviously in English, for example Burgerim is from the English word burger and their logo is in English (https://d2pnvdnftoj8zk.cloudfront.net/wp-content/uploads/sites/9/2018/01/17034317/blogo1.svg).

Some other brands are more confusing and I tried to use what make more sense. It seems that I was wrong with at least one brand, I will try to look how they are mapped in OSM again

Reply:
Thanks @yotamN  - if their logo is in English you are correct to use English in the name tag 👍